client/components/credit-card/stored-card.scss

```diff
@@ -0,0 +1,46 @@
+// Credit Card Payment Box
```

Comment:
Let's add a `/** @format */` to the top so prettier can handle formatting

Reply:
Did we need this comment btw? It should be somewhat explanatory from the component structure.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/24485/commits/380a53f0c4c8f19aa18bd19e66ce6898f20182cb